modules/dynamic-plugins/ref-community-plugins.adoc

```diff
@@ -10,267 +10,10 @@
 Details on how {company-name} provides support for bundled community dynamic plugins are available on the https://access.redhat.com/policy/developerhub-support-policy[Red Hat Developer Support Policy] page.
 ====
 
-{product-very-short} includes the following 50 community plugins:
+{product-very-short} includes the following %%COUNT_3%% community plugins:
```

Comment:
%%COUNT_3%% is weird

Reply:
@themr0c This script to generate plugins content is being reviewed by Nick.